### ISSUE TITLE
implicit namespace packages are missing in `subpackages` template variable

### ISSUE THREAD

**Issue by niosHD** [goerz/better-apidoc#11]
The sub package filter in the `create_package_file` function also removes implicit namespace packages (i.e., https://github.com/goerz/better-apidoc/blob/e1e4c59f709bb2f5d0b2a8f2515f9ad4e63a0bf2/better_apidoc.py#L453).

The issue is apparently already fixed in the upstream apidoc tool which performs the filtering using `shall_skip` (see https://github.com/sphinx-doc/sphinx/blob/b135e7ce4a919f73c04e376a14ce597b0f160cac/sphinx/ext/apidoc.py#L128-L129). Could you adopt this change into better-apidoc?

Furthermore, given that other changes like, for example, exclude support for `shall_skip` have been added upstream, I wonder if you plan to rebase better-apidoc on a more recent version of sphinx-apidoc in general.

**Comment by goerz** [comment]
Yes, I am planning to rebase on the latest version at some point, but I don't know yet when I'll get around to it.
